scripts/gitlint/rules.py

```diff
@@ -31,6 +31,7 @@ def validate(self, line, _commit):
             'fuzz',
             'github',
             'gitignore',
+            'gitlint',
```

Comment:
This was just added from #5963. Can you please rebase? Thanks.

Reply:
Done, thanks.